net/packet/af_packet.c

```diff
@@ -1731,7 +1731,10 @@ static int fanout_add(struct sock *sk, u16 id, u16 type_flags)
 	}
 
 out:
-	kfree(rollover);
```

Comment:
This change should be discarded as it reverts part of f50e9c872cb5 in 4.4.106 (acf47e76937e)